Cargo.toml

```diff
@@ -7,10 +7,9 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bitcoincore-rpc = "0.12.0"
+bitcoincore-rpc = "0.13.0"
 #bitcoincore-rpc = {git="https://github.com/rust-bitcoin/rust-bitcoincore-rpc"}
 bitcoin-wallet = "1.1.0"
-bitcoin = "0.25"
```

Comment:
On further thoughts.

I am not sure if its a good idea to use `bitcoin` from a derived crate like `bitcoincore-rpc`, even though it might expose it publicly.

We will be using `bitcoin` quite heavily through out the project, we might wanna move up in our bitcoin version (or add extra feature we need in `bitcoin)`. For example encoding/decoding tests require `rand` feature of `bitcoin`. 

I think because we might wanna use `bitcoin` in all of its capability, its better to use our own dependency of bitcoin, and not rely on the exposed dependency of derived crates.

So specific to this case, i think duplicating `bitcoin` dependency is better than latter running into weird cargo dep issues.

Reply:
Also, later if we decide to use some other bitcoin derived crates like `rust-miniscript` and if it has a different `bitcoin` dep version, cargo will be duplicating the `bitcoin` deps anyway.

Reply:
> We will be using bitcoin quite heavily through out the project, we might wanna move up in our bitcoin version (or add extra feature we need in bitcoin).

Right, I would think it's pretty easy to add back the `bitcoin` dependency when that need comes up, and/or make a PR upstream to `bitcoincore_rpc` to add a feature(s) that expose `bitcoin` features. Currently the need isn't in the code base, and this PR cleans up an unnecessary duplicate dependency.

> For example encoding/decoding tests require rand feature of bitcoin

That can be handled with a test dependency that doesn't affect the main build, but I get your point. What do you think of the idea I mention above?

> Also, later if we decide to use some other bitcoin derived crates like rust-miniscript and if it has a different bitcoin dep version, cargo will be duplicating the bitcoin deps anyway.

That's a hypothetical problem that can be handled when it comes up. The problem doesn't exist in the current code base.